### DISCUSSION TITLE
Increase Snap Area (setting for mouse distance from edge of screen)

### DISCUSSION THREAD

**Discussion by plasmatech8** [rxhanson/Rectangle#668] [Ideas]
When I click and drag a window to the side of the screen, it can be difficult to snap the window without being really careful with cursor placement.

* When I move the mouse+window to the top of the screen:
  *  I need move the cursor slowly and carefully, to place the cursor on the top of the screen, and not to go too far upwards too quickly or else I will enter Mission Control instead of full-screen snap.
* When I have a monitor on my left and drag the window to the left:
  *  I need move the cursor slowly and carefully, to place the mouse cursor onto the edge of the screen, and not go too far or else the window will move to the next monitor.

It would be really nice if we can have a slider to configure snap area/distance so that can put the cursor a further distance from the edge of the screen to activate window snapping.

This way we can be more lazy and fling windows to areas on the edge of the display without concentrating on cursor position.

**Comment by plasmatech8** [comment]
It might also nice to have a visual snap-area tool which highlights area of the screen where snapping occurs when you want to configure it - and perhaps you can manually configure different areas to do different window snapping. This is might be useful for monitor setups. 

But it is more important to allow increasing the area of snap locations for the cursor so that it is easier to snap windows, imo.

**Comment by rxhanson** [comment]
Is this what you are looking for? 

https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#changing-the-margin-for-the-snap-areas

I’m open to pull requests if anyone wants to add things like this into the UI.

**Reply by plasmatech8** [reply]
Oh yeah, that's exactly what I was looking for. I should have read through the repo properly. Thanks.
